cmd/occollector/app/builder/builder.go

```diff
@@ -55,6 +56,9 @@ func Flags(flags *flag.FlagSet) {
 	flags.Bool(loggingExporterFlg, false, "Flag to add a logging exporter (combine with log level DEBUG to log incoming spans)")
 	flags.Bool(useTailSamplingAlwaysSample, false, "Flag to use a tail-based sampling processor with an always sample policy, "+
 		"unless tail sampling setting is present on configuration file.")
+	flags.Uint(memBallastFlag, 0,
+		fmt.Sprintf("Flag to specify size of memory (MiB) ballast to set. Ballast is not used when this is not specified. "+
+			"default settings: 0"))
```

Comment:
nit: I think that "default settings: 0" is redundant with the previous info. Perhaps add "This can reduce GC cpu usage.", but more important: a quick explanation of this flag on the README.md (can be done in a separate PR).

Reply:
Tracking this via https://github.com/open-telemetry/opentelemetry-service/issues/57